include/deal.II/lac/la_parallel_vector.templates.h

```diff
@@ -28,6 +28,10 @@
 #include <deal.II/lac/vector_operations_internal.h>
 
 #include <Kokkos_Core.hpp>
+#if !DEAL_II_KOKKOS_VERSION_GTE(4, 5, 0)
+// for reduction_identity<bool>
+#  include <Kokkos_ReductionIdentity.hpp>
+#endif
```

Comment:
That header exists since 4.0. but I would be surprised if you actually need to include it explicitly since we aren't even using `reduction_identity`.

Reply:
I tried to solve this error message:

```
2025-10-28T16:44:44.1542981Z /__w/dealii/dealii/source/lac/la_parallel_vector.cc:47:20:   required from here
2025-10-28T16:44:44.1543852Z /usr/include/trilinos/Kokkos_Parallel_Reduce.hpp:287:47: error: incomplete type ‘Kokkos::reduction_identity<bool>’ used in nested name specifier
2025-10-28T16:44:44.1544414Z   287 |     val = reduction_identity<value_type>::land();
```
butt adding that header doesn't work for exactly that reason.